shell/common/shell_unittests.cc

```diff
@@ -1277,12 +1277,14 @@ TEST_F(ShellTest,
   DestroyShell(std::move(shell));
 }
 
+// TODO(https://github.com/flutter/flutter/issues/95751): Disabled due to
```

Comment:
Could you please file a new issue to re-enable these tests and link it here? That way we still have an open issue to re-enable the tests. Thanks.

Reply:
done